index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>freeCodeCamp Survey Form</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <h2 id="title">freeCodeCamp Survey Form</h2>
```

Comment:
- [ ] I think it would be better to use a < h1 > on your project title to replace < h2> 😎😎 And If you want that google (and other search engines) to know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.